code/modules/admin/admin_verbs.dm

```diff
@@ -585,7 +585,7 @@ var/list/admin_verbs_mentors = list(
 
 /client/proc/togglebuildmodeself()
 	set name = "Toggle Build Mode Self"
-	set category = "Special Verbs"
+	set category = "Server"
```

Comment:
This is `Sever` how?

Reply:
Affects server // moreso debug/event related

Reply:
> Affects server
If use that logic then any verb affects the server, it was in Special Verbs because it's unique and doesn't fit anywhere else